fastlane/lanes/build.rb

```diff
@@ -359,7 +359,7 @@ def build_and_upload_prototype_build(scheme:, output_app_name:, appcenter_app_na
     configuration = 'Release-Alpha'
 
     # Get the current build version, and update it if needed
-    version_config_path = File.join(PROJECT_ROOT_FOLDER, 'config', 'Version.internal.xcconfig')
+    version_config_path = File.join(PROJECT_ROOT_FOLDER, 'config', 'Version.public.xcconfig')
```

Comment:
Similar to the comments above, the public version file's release version (e.g. 24.4) is the same as the internal's, so this change won't affect anything. The build numbers were different between the public and internal, but for these prototype builds, their build numbers are generated on-demand anyway, superseding the existing build number.

Reply:
I think this should work, as long as the lane is `import`ed instead of `require`d (`import` gives access to constants) after `PUBLIC_VERSION_FILE` is defined

```suggestion
    version_config_path = PUBLIC_VERSION_FILE
```